### PR TITLE
Docs for Dropbox upload/download file

### PR DESCRIPTION
**Pull request by zweedao**

### CODE REVIEW

**Comment by zweedao** [comment]
@graceliching can you help

File: connectors/dropbox/action-download-file.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Workato connector - Dropbox - Download file
+date: 2019-01-31 06:00:00 Z
+---
+
+# Dropbox action - Download file
+Download contents of a file. File contents can be used in subsequent steps of the recipe to upload the file into other applications.
```
Comment (graceliching) [inline connectors/dropbox/action-download-file.md]: This action downloads contents of a file. The file contents output from this recipe step can be used in subsequent steps of the recipe to upload the file into other applications.

File: connectors/dropbox/action-download-file.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+The following is an example of uploading a file using `File contents`:
```
Comment (graceliching) [inline connectors/dropbox/action-download-file.md]: The following image is an example of how the output pill from the 'Download file' action can be used. Here, the data pill, `File contents`, is chosen from the data tree in  the 'Upload file using file contents' action:

**Comment by graceliching** [comment]
few comments @zweedao

**Comment by zweedao** [comment]
done @graceliching

**Pull request merged**